backend/routes/db.py

```diff
@@ -343,12 +342,19 @@ def _cset_members_items(codeset_ids: Union[str, None] = Query(default=''), ) ->
     codeset_ids: List[int] = parse_codeset_ids(codeset_ids)
     return get_cset_members_items(codeset_ids)
 
-
 @router.get("/db-refresh")
-def db_refresh_route() -> Response:
-    """Triggers refresh of the database"""
-    response: Response = call_github_action('refresh-db')
-    return response
+def db_refresh_route():
```

Comment:
### Change back to GitHub action?
@Sigfried I thought this change to call backend directly rather than GH action. I'm going to merge this now, as we've decided. But I think maybe we should go back to the GH action for the following reasons:

1. Easier to check logs
If there's a problem, I can go to the actions tab and find easily. Finding on azure takes many more clicks, and then I have to scroll up to find where the refresh got logged, and it may be mixed with logs for other requests.
2. Almost always won't increase speed of refresh
This was supposed to be the only benefit of calling it directly.
If someone creates a cset and clicks the button, it doesn't matter that our backend is faster than a GH action starting up, since it will take 20-45 minutes for that cset to be ready anyway. so effectively this is not faster, except for fetching csets that are not brand new, which (i) is not the primary thing people are using the button for, and (ii) is unlikely to happen w/ a fast refresh rate.
3. Harder to make changes
If we want to make changes to the refresh, we have to redeploy the whole app to make that work, rather than pushing to develop.
4. Uses more server resources.
5. Possible server stability issues
I'm not sure, but I wonder if there is some edge case where an error that happens during the refresh, or other unanticipated side effects, could have an effect on performance or stability of the web server.
This is on top of the

Reply:
How about this: when the api call is made, if backend is running locally, run directly; if on prod, run GH action; if on dev, flip a coin :)  

The button call's `db-refresh` route in any case. Just add some logic [here](https://github.com/jhu-bids/TermHub/blob/3ee2ee4b18a29724667c01986ebea2d565438e48/backend/routes/db.py#L359-L361)

Reply:
I was thinking about that, except I think dev should call the GH action as well.

But for local, I don't know if it even makes sense. It will be the same as running the refresh script with no params, which you can do via right click + debug. So not the biggest priority for me.

Thanks for reading my points. I'll do this after some of the more critical db refresh priorities.

If for some reason members start coming through the enclave API faster, we can change it back as speed will matter more in that situtaion.